modules/op-resolver-community-tasks-ref.adoc

```diff
@@ -0,0 +1,411 @@
+        - name: helm_image
+          value: 'docker.io/lachlanevenson/k8s-helm@sha256:5c792f29950b388de24e7448d378881f68b3df73a7b30769a6aa861061fd08ae'
+        - name: upgrade_extra_params
+          value: ''
+      taskRef:
+        kind: Task
+        name: helm-upgrade-from-source
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+  #...
+----
+
+.Supported parameters for the `helm-upgrade-from-source` community task
+[cols="1,2,1",options="header"]
+|===
+| Parameter | Description | Default value
+|`charts_dir` |The directory in the source workspace that contains the Helm chart. |
+|`release_version` |The Helm release version in semantic versioning format. | `v1.0.0`
+|`release_name` |The Helm release name. |`helm-release`
+|`release_namespace` |The Helm release namespace. |`""`
+|`overwrite_values` |The configuration parameters to overwrite, comma separated. For example: `autoscaling.enabled=true,replicas=1`|`""`
+|`values_file` |The file with configuration parameters to be used. |`values.yaml`
+|`helm_image` |The Helm image to be used. |`docker.io/lachlanevenson/k8s-helm@sha256:5c792f29950b388de24e7448d378881f68b3df73a7b30769a6aa861061fd08ae`
+|`upgrade_extra_params` |Extra parameters passed for the Helm upgrade command.|`""`
+|===
+
+.Supported workspaces for the `helm-upgrade-from-source` community task
+[cols="1,3",options="header"]
+|===
+| Workspace | Description
+| `source` | The workspace that contains the Helm chart.
+|===
+
+[discrete]
+[id="op-ctaskref-jib-maven_{context}"]
+== jib-maven
+
+The `jib-maven` community task builds Java, Kotlin, Groovy, and Scala sources into a container image by using the Jib tool for Maven projects. 
+
+.Example usage of the `jib-maven` community task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: jib-maven
+spec:
+  tasks:
+    - name: jib-maven
+      params:
+        - name: IMAGE
+          value: example_image
+        - name: MAVEN_IMAGE
+          value: 'registry.redhat.io/ubi9/openjdk-17@sha256:78613bdf887530100efb6ddf92d2a17f6176542740ed83e509cdc19ee7c072d6'
+        - name: DIRECTORY
+          value: .
+        - name: CACHE
+          value: empty-dir-volume
+        - name: INSECUREREGISTRY
+          value: 'false'
+        - name: CACERTFILE
+          value: service-ca.crt
+      taskRef:
+        kind: Task
+        name: jib-maven
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+  #...
+----
+
+.Supported parameters for the `jib-maven` community task
+[cols="1,2,1",options="header"]
+|===
+| Parameter | Description | Default value
+|`IMAGE` |Name of the image to build. |
+|`MAVEN_IMAGE` |Maven base image. |`registry.redhat.io/ubi9/openjdk-17@sha256:78613bdf887530100efb6ddf92d2a17f6176542740ed83e509cdc19ee7c072d6`
+|`DIRECTORY` |The{nbsp}directory{nbsp}containing{nbsp}the{nbsp}app,{nbsp}relative{nbsp}to the source repository root. |`.`
```

Comment:
Remove "The" here and in the other parameters, I think we don't normally use an article at the start of a parameter description in the table (compare to how the non-community tasks are done)